conans/client/tools/system_pm.py

```diff
@@ -140,6 +154,9 @@ def installed(self, package_name):
 
 
 class YumTool(object):
+    def add_repository(self, repository, repo_key=None):
+        raise ConanException("YumTool::add_repository not implemented (see #3385)")
```

Comment:
Not sure about the error message with the #3385. WDYT @memsharded?

Reply:
Yes, I wouldn't use it here. Because #3385 will be closed as implemented, too, which might be confusing. Too much coupled.